pkg/render/manager.go

```diff
@@ -139,7 +139,7 @@ type ManagerConfiguration struct {
 	// KeyPair used for establishing mTLS tunnel with Guardian.
 	TunnelSecret certificatemanagement.KeyPairInterface
 
-	// TLS KeyPair used by Voltron within the cluster when operating as a management cluster.
+	// TLS KeyPair used by Voltron/EsProxy within the cluster when operating as a management cluster.
```

Comment:
I think we need to improve this comment more - it's not super clear what the different uses are for this, and I believe as part of this PR it's no longer just used in management clusters.

Is this righ? 

```
// TLS KeyPair used by both Voltron and es-proxy, presented by each as part of the mTLS handshake with other services within the cluster. This is used in both management and standalone clusters. 
```